telegram_service.py

```
@@ -0,0 +1,276 @@
+            return
+        import httpx
+
+        def _send():
+            for chat_id in self.telegram_chat_ids:
+                try:
+                    with httpx.Client(timeout=5.0) as client:
+                        client.post(
+                            f"https://api.telegram.org/bot{self.telegram_token}/sendMessage",
+                            json={"chat_id": chat_id, "text": message, "parse_mode": "Markdown"},
+                        )
+                except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)